app/models/tracked_email.rb

```diff
@@ -15,7 +15,7 @@ def mail_to_send
     raise NotImplementedError
   end
 
-  def send!
+  def send!(*)
```

Comment:
This is splat operator/naked splat right? It lets you pass any number of arguments to super but we aren't calling super at any point are we? 

I'm just trying to understand why it's needed if we're overriding the method in invite_email_ect and invite_email_mentor .